### PR TITLE
Working branch for weather app

### PR DESCRIPTION
**Pull request by arslanbisharat**
This is the 5th project of the Main Javascript curriculum at [Microverse](https://www.microverse.org/) - @microverseinc

* The objective is to build a page with interactivity using everything learn about **Objects**, **Factory Functions**, The **Module Pattern**
* The project was completed using Javascript language.

### CODE REVIEW

File: src/DOMelements.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  menuToggle("menu-overlay");
+});
+
+// celcius
+let metricButton = document.querySelector(".metric");
+// fahrenheit
+let imperialButton = document.querySelector(".imperial");
+let unitsButtons = [metricButton, imperialButton];
+
+function getUnit() {
```
Comment (suretrust) [inline src/DOMelements.js:39]: Use es6 in place of es5 `function`.
```suggestion
const getUnit = () => {
```

File: src/autocomplete.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// PLACES
+export let placesAutocomplete = places({
```
Comment (suretrust) [inline src/autocomplete.js]: Since this is not changing, use `const` instead.
```suggestion
export const placesAutocomplete = places({
```

**Review by suretrust — CHANGES_REQUESTED** [review]
Hi @arslanbisharat, 👋 

### Great job thus far! 🙌 Please see my comments. 👇 

- [ ] Ensure the linter has no errors.

* [ ] It should have a toggle control to determine if the result will be shown in Celsius or Fahrenheit. I understand that you already have both separately but the `toggle` feature is missing, toggle to change from Celsius to Fahrenheit and vice versa. Also, the application should make different requests for each temperature type. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app)).
* [x] It uses ES6 syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622133/)).
  * [x] usage of arrow functions instead of the `function` keyword.
  * [x] proper usage of `const` and `let` keywords.
> I quoted some examples of what these mean in your code below. ⬇️ 

**Kindly submit another CRR once these issues have been resolved.** _Happy coding!_ 🎉

**Review by shloch — APPROVED** [review]
the website animation is great

accomplished

We're good to go ...

**Pull request merged**